### ISSUE TITLE
Microsoft-provided Github action link changed to third-party action with misleading commit message

### ISSUE THREAD

**Issue by jikuja** [MicrosoftDocs/azure-dev-docs#886]
https://github.com/MicrosoftDocs/azure-dev-docs/commit/ce17a47dd75157a65da817f57ec352dee5e59c35

Above mentioned commit changed deprecated Microsoft-provided Github key vault action to third-party action. To wording of the commit raises multiple questions:

* Action is not from verified creator. Is that suitable for documentation links?
* Does the action has feature parity with deprecated action or was it chosen by name that is enough similar with deprecated action?
* Was the change coordinated with authors of https://github.com/Azure/get-keyvault-secrets?
* Is the content sync with https://docs.microsoft.com/en-us/azure/developer/github/github-key-vault?
* TLDR: broken ink was not fixed. The linked action was changed.

Related issue: https://github.com/Azure/get-keyvault-secrets/issues/33

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4da18184-e5e5-e17c-614c-3361751014a4
* Version Independent ID: 43035cf9-0cc8-e649-490c-b223af47d991
* Content: [What is GitHub Actions for Azure?](https://docs.microsoft.com/en-us/azure/developer/github/github-actions)
* Content Source: [articles/github/github-actions.md](https://github.com/MicrosoftDocs/azure-dev-docs/blob/main/articles/github/github-actions.md)
* Service: **azure**
* GitHub Login: @N-Usha
* Microsoft Alias: **ushan**

**Comment by juliakm** [comment]
Thanks @jikuja. The Azure Key Vault action is no longer maintained. We will make it clearer that this is a 3rd party action.
